package_drupal_script.sh

```diff
@@ -144,11 +144,11 @@ cp ${REPO_DIR}/example.gitignore ${STAGING_DIR}/sprint/drupal8/.gitignore
 
 echo "Running composer install --quiet"
 composer install --quiet
+popd >/dev/null
 
 # Copy licenses and COPYING notice.
-cp -r ${REPO_DIR}/licenses "$STAGING_DIR/"
-cp ${REPO_DIR}/COPYING "$STAGING_DIR/"
-popd >/dev/null
```

Comment:
Whoops. Maybe a good test case to assert the package contents?

Reply:
I added assertion on these items. Will keep thinking about the rest of the contents, but they'd all cause other failures so it doesn't seem as bad.